mycroft/client/speech/main.py

```diff
@@ -56,6 +56,10 @@ def handle_utterance(event):
     ws.emit(Message('recognizer_loop:utterance', event))
 
 
+def handle_unknown():
+    ws.emit(Message('mycroft.mycspeech.recognition.unknown'))
```

Comment:
Is "mycspeech" intentional?  Looks like a search/replace error.

Reply:
Lol, no idea how I failed at typing that much. Fixed now.